public/index.html

```diff
@@ -39,5 +39,7 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+
+    <div class='panel'></div>
```

Comment:
buraya niye html elemani ekledin?

Reply:
dropdown componentinin 29. satırında görebilirsin ne için kullandığımı